baker/Deployer.ts

```diff
@@ -0,0 +1,374 @@
+import * as fs from "fs-extra"
+import * as prompts from "prompts"
+import ProgressBar = require("progress")
+import { execWrapper } from "../db/execWrapper"
+import { spawn } from "child_process"
+import simpleGit, { SimpleGit } from "simple-git"
+import { WriteStream } from "tty"
+import { ProgressStream } from "./ProgressStream"
+
+const TEMP_DEPLOY_SCRIPT_SUFFIX = `.tempDeployScript.sh`
+
+enum DeployTarget {
+    staging = "staging",
+    hans = "hans",
+    playfair = "playfair",
+    jefferson = "jefferson",
+    nightingale = "nightingale",
+    explorer = "explorer",
+    exemplars = "exemplars",
+    tufte = "tufte",
+    roser = "roser",
+}
+
+interface DeployerOptions {
+    owidGrapherRootDir: string
+    userRunningTheDeploy: string
+    target: DeployTarget
+    skipChecks?: boolean
+    runChecksRemotely?: boolean
+}
+
+const OWID_STAGING_DROPLET_IP = "165.22.127.239"
+const OWID_LIVE_DROPLET_IP = "209.97.185.49"
```

Comment:
```suggestion
const OWID_STAGING_SERVER_IP = "165.22.127.239"
const OWID_LIVE_SERVER_IP = "209.97.185.49"
```

... so as to not get too into it with the DigitalOcean terminology

Reply:
I am a digital ocean fanboy. While I agree with you, and wouldn't object if someone else makes this change, I cannot personally bring myself to do it.